Resources/Prototypes/Entities/Mobs/NPCs/funnyayadoll.yml

```diff
@@ -0,0 +1,89 @@
+- type: entity
+  name: poltergeist
```

Comment:
Just `poltergeist` suggests to me a ghost, when this is a doll. Maybe a different name?

Reply:
a poltergeist was the original idea but it changed overtime but i will look into it